finagle-netty4/src/main/scala/com/twitter/finagle/netty4/channel/ServerBridge.scala

```diff
@@ -11,7 +11,9 @@ import io.netty.channel.ChannelHandler.Sharable
 private[netty4] class ServerBridge[In, Out](
     transportFac: Channel => Transport[In, Out],
     serveTransport: Transport[In, Out] => Unit)
-  extends ChannelInitializer[Channel] {
+  extends ChannelInboundHandlerAdapter {
+
+  override def handlerAdded(ctx: ChannelHandlerContext): Unit = {}
```

Comment:
OMG, have no idea how it is landed into PR.

I think it is a good idea! Updated PR